src/libcore/sync/atomic.rs

```diff
@@ -927,6 +927,12 @@ impl<T> AtomicPtr<T> {
     }
 }
 
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
This is wrong as it implies the implementation has existed since 1.0.0. You need to make a new stable feature.

Reply:
Ahh, that makes sense now — it complained about `(feature = "rust1", since = "1.22.0")`.